src/platform_impl/windows/event_loop.rs

```diff
@@ -596,7 +596,19 @@ lazy_static! {
 fn create_event_target_window() -> HWND {
     unsafe {
         let window = winuser::CreateWindowExW(
-            winuser::WS_EX_NOACTIVATE | winuser::WS_EX_TRANSPARENT | winuser::WS_EX_LAYERED,
+            winuser::WS_EX_NOACTIVATE
+                | winuser::WS_EX_TRANSPARENT
+                | winuser::WS_EX_LAYERED
+                // WS_EX_TOOLWINDOW prevents this window from ever showing up in the taskbar, which
+                // we want to avoid. If you remove this style, this window won't show up in the
+                // taskbar *initially*, but will eventually magically appear in the taskbar.
+                //
+                // Now, I wasn't able to figure out what *causes* the window to eventually show up
+                // in the taskbar, since it does so without warning, after several hours, with no
+                // obvious code-visible state changes to this window. I'd guess that some win32
+                // state change gets triggered at some point, but I'd need to be able to debug the
+                // windows shell directly to figure that out and I can't do that.
```

Comment:
```suggestion
                // WS_EX_TOOLWINDOW prevents this window from ever showing up in the taskbar, which
                // we want to avoid. If you remove this style, this window won't show up in the
                // taskbar *initially*, but it can show up at some later point. This can sometimes
                // happen on its own after several hours have passed, although this has proven
                // difficult to reproduce. Alternatively, it can be manually triggered by killing
                // `explorer.exe` and then starting the process back up.
                // It is unclear why the bug is triggered by waiting for several hours.
```